presto-docs/src/main/sphinx/release/release-329.rst

```diff
@@ -0,0 +1,50 @@
+=========================
+Release 329 (XXX 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect result of ``last_day_of_month`` function for first day of month. (:issue:`2452`)
+* Handle common disk failures during the spill. When on of disks got broken, it Presto will use another
+  one without failing the query. (:issue:`2444`)
+* Reduce number of metadata queries when querying ``information_schema``. (:issue:`2488`)
+* Limit broadcasted table size to 100MB by default when using the ``AUTOMATIC`` join reordering strategy. (:issue:`2527`)
+
+Security Changes
+----------------
+
+* Add ``ldap.bind-dn`` and ``ldap.bind-password`` LDAP properties to allow LDAP authentication
+  access LDAP server using service account. (:issue:`1917`)
+
+CLI Changes
+-----------
+
+* Allow inspecting network requests. (:issue:`2524`)
+
+Hive Connector Changes
+----------------------
+
+* Fix incorrect data returned when using S3 Select on uncompressed files. In our testing, S3 Select
+  was apparently returning incorrect results when reading uncompressed files, so S3 Select is disabled
+  for uncompressed files. (:issue:`2399`)
+* Fix incorrect data returned when using S3 Select on a table with ``skip.header.line.count`` or
+  ``skip.footer.line.count`` property. S3 Select API does not support skipping footers or more than one
+  line of a header.  In our testing, S3 Select was apparently sometimes returning incorrect results when
+  reading a compressed file with header skipping, so S3 Select is disabled when any of these table
+  properties is set to non-zero value. (:issue:`2399`)
+* Fix ``INSERT`` or ``CREATE TABLE ... AS`` query failure when one of the inserted ``REAL`` or ``DOUBLE`` values
+  is infinite or ``NaN``. (:issue:`2471`)
+* Add explicit support for conversion from Parquet decimal column type to Presto integer column types. (:issue:`2451`)
+* Add validation of Parquet decimal column type scale. (:issue:`2451`)
```

Comment:
@sopel39 what's end-user visible effect?

Reply:
Based on the tests, I think this can be
```
* Validate the scale of decimal types when reading Parquet files. This prevents
  incorrect results when the decimal scale in the file does not match the declared
  type for the table or partition. (:issue:`2451`)

Reply:
Actually, this is part of the change above that allows reading these at all, so I'm not sure that it was possible to get incorrect results previously. Maybe it's correct to remove this?

Reply:
Maybe something like this?
`Prevent incorrect reading of decimal values from Parquet decimal columns, which don't match partition level decimal column type`

Reply:
> Actually, this is part of the change above that allows reading these at all, so I'm not sure that it was possible to get incorrect results previously. Maybe it's correct to remove this?

It was possible as long as both decimals were of the same type (e.g short or long). Parquet code was just inserting unscaled values...